docs-site/content/0.25.0/api/vector-search.md

```diff
@@ -310,216 +342,1504 @@ curl "http://localhost:8108/collections/docs/documents" -X POST \
+          "refresh_token": "your_gcp_refresh_token",
+          "client_id": "your_gcp_app_client_id", 
+          "client_secret": "your_gcp_client_secret",
+          "project_id": "your_gcp_project_id"
+        }
+      }
+    }
+  ]
+};
+
+client.collections('products').create(schema);
+
+```
+  </template>
+
+  <template v-slot:PHP>
+
+```php
+$schema = [
+  "name" => "products",
+  "fields" => [
+    [
+      "name" => "product_name",
+      "type" => "string"
+    ],
+    [
+      "name" => "embedding",
+      "type" => "float[]",
+      "embed" => [
+        "from" => [
+          "product_name"
+        ],
+        "model_config" => [
+          "model_name" => "gcp/embedding-gecko-001",
+          "access_token" => "your_gcp_access_token",
+          "refresh_token" => "your_gcp_refresh_token",
+          "client_id" => "your_gcp_app_client_id", 
+          "client_secret" => "your_gcp_client_secret",
+          "project_id" => "your_gcp_project_id"
+        ]
+      }
+    ]
+  ]
+];
+
+$client->collections->create($schema);
+
+```
+  
+  </template>
+  <template v-slot:Python>
+
+```py
+schema = {
+  "name": "products",
+  "fields": [
+    {
+      "name" : "product_name",
+      "type" : "string"
+    },
+    {
+      "name" : "embedding",
+      "type" : "float[]",
+      "embed": {
+        "from": [
+          "product_name"
+        ],
+        "model_config": {
+          "model_name": "gcp/embedding-gecko-001",
+          "access_token": "your_gcp_access_token",
+          "refresh_token": "your_gcp_refresh_token",
+          "client_id": "your_gcp_app_client_id", 
+          "client_secret": "your_gcp_client_secret",
+          "project_id": "your_gcp_project_id"
+        }
+      }
+    }
+  ]
+}
+
+client.collections.create(schema)
+```
+
+  </template>
+
+  <template v-slot:Shell>
+     
+```bash
+curl -X POST \
+  'http://localhost:8108/collections' \
+  -H 'Content-Type: application/json' \
+  -H 'X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}' \
+  -d '{
+  "name": "products",
+  "fields": [
+    {
+      "name": "product_name",
+      "type": "string"
+    },
+    {
+      "name": "embedding",
+      "type": "float[]",
+      "embed": {
+        "from": [
+          "product_name"
+        ],
+        "model_config": {
+          "model_name": "gcp/embedding-gecko-001",
+          "access_token": "your_gcp_access_token",
+          "refresh_token": "your_gcp_refresh_token",
+          "client_id": "your_gcp_app_client_id", 
+          "client_secret": "your_gcp_client_secret",
+          "project_id": "your_gcp_project_id"
+        }
+      }
+    }'
+
+```
+   </template>
+
+</Tabs>
+
+### Remote Embedding API parameters
+
+You can use any of the following parameters to fine-tune how API calls are made to remote embedding services:
+
+#### During Search
+
+| Parameter                     | Description                                                                                               |
```

Comment:
Could you add a new column here called `Default` and add the default value for each parameter?